praw/models/user.py

```diff
@@ -172,3 +172,18 @@ def subreddits(
         return ListingGenerator(
             self._reddit, API_PATH["my_subreddits"], **generator_kwargs
         )
+
+    def trusted(self) -> List["praw.models.Redditor"]:
+        """Return a RedditorList of trusted Redditors.
+
+        To display the names of your trusted users and the times at which you decided to trust
+            them, try:
+
+        .. code-block:: python
+
+            trusted_users = reddit.user.trusted()
+            for user in trusted_users:
+                print(f"User: {user.name}, time: {user.date}")
+
+        """
+        return self._reddit.get(API_PATH["trusted"])
```

Comment:
Just wanted to confirm, this returns all of the users regardless of how many there is, i.e., not paginated?

Reply:
That's a good question. The sibling method [blocked](https://github.com/praw-dev/praw/blob/master/praw/models/user.py#L59) also returns a  "UserList", as [shown here](https://github.com/praw-dev/praw/blob/master/tests/integration/cassettes/TestUser.test_blocked_fullname.json#L137). Unlike "Listing"s, there are no "before" and "after" params included. I believe that it isn't paginated, which does cause some [problems](https://www.reddit.com/r/redditdev/comments/mz17w9/i_have_too_many_friends_consistent_503_error/).

Reply:
I see, I was just making sure it wasn't something that needed a `ListingGenerator`.